backend/uclapi/libcal/views.py

```diff
@@ -306,13 +306,11 @@ def get_bookings(request, *args, **kwargs):
     if "data" in data:
         data["bookings"] = data.pop("data")
         if isinstance(data["bookings"], list):
-            for booking in data["bookings"]:
-                # Strip personal data from response
-                booking.pop('email', None)
-                booking.pop('first_name', None)
-                booking.pop('last_name', None)
-                booking.pop('book_id', None)
-                booking.pop('check_in_code', None)
+            for i in range(len(data["bookings"])):
+                # Ensure we don't leak personal data, only allow the following fields
+                allowed_fields = ['eid', 'cid', 'lid', 'from_date', 'to_date', 'created', 'status',
```

Comment:
We could perhaps have this be a settings entry, either in .env or in settings.py?

Reply:
I think this is probably okay, it is just for this endpoint and we don't want to pollute our .env or settings file.